tern/analyze/docker/dockerfile.py

```diff
@@ -333,3 +334,47 @@ def write_locked_dockerfile(dfile, destination=None):
         file_name = constants.locked_dockerfile
     with open(file_name, 'w') as f:
         f.write(dfile)
+
+
+def check_multistage_dockerfile(dfobj):
+    """Given a dockerfile object, return the index(es) of FROM line(s)
+    in the dfobj structure."""
+    from_lines = []
+    for idx, st in enumerate(dfobj.structure):
+        if st['instruction'] == 'FROM':
+            from_lines.append(idx)
+    return from_lines
```

Comment:
I have removed True/False return value and use `from_lines` here since there are possibly more than one FROM line.